app/serializers/group_serializer.rb

```diff
@@ -57,4 +57,8 @@ def has_discussions
   def has_multiple_admins
     object.admins.count > 1
   end
+
+  def cover_photo
```

Comment:
If it's using a default cover photo, which reach out to the DefaultCoverPhoto of the group.